src/binaries/BinariesManager.ts

```diff
@@ -19,25 +18,13 @@ export class BinariesManager {
                                      accountContextManager: AccountContextManager,
                                      expectedCLIVersion: string
     ): IBinariesUtility {
-        if (this.useBinariesUtilityV2()) {
-            logger.trace(TelemetryEvent.BinariesUtility_VersionV2);
+        if (this.useBinariesUtility()) {
+            logger.trace(TelemetryEvent.BinariesUtility_Version);
             return new BinariesUtilityV2(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
         }
-
-        logger.trace(TelemetryEvent.BinariesUtility_VersionV1);
-        return new BinariesUtility(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
     }
 
-    private static useBinariesUtilityV2(): boolean {
-        const useBinaryUtilityEnvironmentVariable: string = process.env.BRIDGE_BINARYUTILITYVERSION;
```

Comment:
💡  did you see this message https://github.com/Azure/vscode-bridge-to-kubernetes/pull/10#discussion_r987439099

I wonder if it is worth to get @daniv-msft  eye here as well for this `BRIDGE_BINARYUTILITYVERSION` variable before it gets removed etc.

Later lets generate visa and get it tested from PMs. Thank you.

Reply:
No problem from me with dropping BRIDGE_BINARYUTILITYVERSION: this is only used for this scenario and should be safe to remove.